core/modules/menu/js/menu.admin.js

```diff
@@ -19,6 +19,62 @@ Backdrop.behaviors.menuAdminFieldsetSummaries = {
   }
 };
 
+Backdrop.behaviors.menuEditItemParents = {
+  attach: function (context, settings) {
+    $('.form-item-parent-menu', context).once('form-item-parent-menu', function () {
+      // Get the menu parent options from settings.
+      var menuOptions = settings.menu_edit_item_parents;
+      // Get the current menu name from settings.
+      var menuName = settings.menu_edit_item_menu;
+      // Always move the menu selector before the menu parent selector.
+      $('.form-item-parent-menu').insertBefore('#menu-parent-select-wrapper');
+      // On load set the current menu as default in the menu select.
+      Backdrop.menu_edit_update_parent_list(menuName);
+      // Ensure that the menu parent select list is filtered again when any ajax
+      // process runs.
+      $(document).ajaxComplete(function(event, xhr, settings) {
```

Comment:
Method ajaxComplete is deprecated in jQuery. Use `.on( "ajaxComplete", handler )` instead.

See https://api.jquery.com/ajaxComplete-shorthand/

Reply:
I've fixed this. There's one other instance in `installer.project_list.js` BTW.